cli/gh.go

```diff
@@ -8,11 +8,11 @@ import (
 )
 
 // wrap the common gh lib shared with my other tools. splits common GH code from this CLI tool's specific tooling code
-type githubRepo struct {
+type GithubRepo struct {
 	gh.Repo
 }
 
-func (f FlagData) NewRepo() githubRepo {
+func (f FlagData) NewRepo() GithubRepo {
```

Comment:
cli/gh.go:15:29: unexported-return: exported method NewRepo returns unexported type cli.githubRepo, which can be annoying to use (revive)
```
func (f FlagData) NewRepo() githubRepo {
                            ^
```